service-api/module/Application/src/Controller/IdentityController.php

```diff
@@ -564,4 +564,41 @@ public function updateDobAction(): JsonModel
 
         return new JsonModel($response);
     }
+
+    public function updateCpPoIdAction(): JsonModel
+    {
+        $uuid = $this->params()->fromRoute('uuid');
+        $data = json_decode($this->getRequest()->getContent(), true);
+        $response = [];
+        $status = Response::STATUS_CODE_200;
+
+        if (! $uuid) {
+            $status = Response::STATUS_CODE_400;
+            $this->getResponse()->setStatusCode($status);
+            $response = [
+                "error" => "Missing UUID"
+            ];
+            return new JsonModel($response);
```

Comment:
Errors should return `Problem` so that we have a consistent structure for error messages

```suggestion
            return new Problem("Missing UUID");
```